src/main/java/seedu/finance/model/record/Date.java

```diff
@@ -12,7 +12,8 @@
  */
 public class Date {
 
-    public static final String MESSAGE_CONSTRAINTS = "Dates should be of the format dd/mm/yyyy and be a valid date";
+    public static final String MESSAGE_CONSTRAINTS = "Dates should be of the format dd/mm/yyyy and be a valid date"
+            + " that is not later than is not after today's date in local time.";
```

Comment:
.. not later than
or
is not after today's date..

Reply:
oo right, haha sorry for the typo